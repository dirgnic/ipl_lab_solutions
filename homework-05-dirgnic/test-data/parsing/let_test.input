let x = 5;
return x;
