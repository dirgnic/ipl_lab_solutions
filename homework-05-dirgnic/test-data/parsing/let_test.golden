Seq [Let "x" (Num 5) (Res (Num 0))] (Res (Var "x"))
